gen3_tracker/meta/dataframer.py

```diff
@@ -612,6 +612,39 @@ def flattened_specimen(self, specimen: dict, observation_by_id: dict) -> dict:
 
         return flat_specimen
 
+    def flattened_group_members(self) -> Generator[dict, None, None]:
+        """generator that yields fhir group entities, flattening out members.entities.reference"""
+
+        resource_type = "Group"
+        cursor = self.connect()
+
+        # get all groups
+        cursor.execute(
+            "SELECT * FROM resources where resource_type = ?", (resource_type,)
+        )
+
+        # flatten groups into group members
+        for _, _, resource in cursor.fetchall():
+            group = json.loads(resource)
+
+            # flatten group, typed for code readability
+            group_resource: SimplifiedGroup = SimplifiedResource.build(resource=group)
+            simplified_group = group_resource.simplified
+
+            # for each member in a group, yield a group member dict
+            for member_id in group_resource.members:
+                # unique primary key from group and member ids
+                group_member_id = (
+                    simplified_group["id"] + "-" + member_id
```

Comment:
this works, but it's not following the UUID standard that is getting used for the other data. Make a fresh UUID using the 'simplified_group["id"] + "-" + member_id' template that you have here. 

See https://github.com/ACED-IDP/g3t_etl/blob/development/g3t_etl/__init__.py#L64-L75
for how to go about minting a new UUID

Reply:
updated thanks